src/checkqueue.h

```diff
@@ -188,7 +188,7 @@ class CCheckQueue
     }
 
     //! Stop all of the worker threads.
-    void StopWorkerThreads()
+    void StopWorkerThreads() EXCLUSIVE_LOCKS_REQUIRED(!m_mutex)
```

Comment:
Should each of the annotations added in the first commit also have `AssertLockNotHeld`?

Reply:
That feels like overkill, but I'm not sure where to draw the line.

Reply:
Closing as resolved?